protobuf/coreapi/api.proto

```diff
@@ -157,9 +157,23 @@ message ListenResultRequest {
 message ResultData {
   string executionID = 1;             // The unique identifier of the execution.
   string taskKey = 2;                 // The key of the executed task.
-  string outputKey = 3;               // The output's key from the returned task.
-  string outputData = 4;              // The output's data from the returned task, encoded in JSON.
   repeated string executionTags = 5;  // The list of tags associated with the execution
+  oneof result {
```

Comment:
The same should be done to lisent event

Reply:
Can you explain more please?

Reply:
Yep, ListenEvent is also a stream and by analogy, it might return an error. So ListenEventRequest 
should have oneof { Result, Error } as well

Reply:
```proto
message ListenEventRequest {
  message Result {
    string serviceID = 1; 
    string eventFilter = 2;
  }
  oneof result {
    Result result = 3;
    Error error = 4;
  }
}
```

Reply:
I quite agree with hubert:
if the service send a not valid event to the core, should we notify the app as well? it could be good yes.
@antho1404 what do you think about this?

Reply:
As this one is not urgent we should merge this PR to fix an actual bug in the codebase. Let's create an another issue for this one and move the discussion there.

Reply:
moved to https://github.com/mesg-foundation/core/issues/600